Mailer/README.md

```diff
@@ -520,6 +520,112 @@ Response:
 
 ---
 
+#### POST `/api/v1/users/is-user-unsubscribed`
+
+API call that checks if user unsubscribed given newsletter.
+
+##### *Headers:*
+
+| Name | Value | Required | Description |
+| --- |---| --- | --- |
+| Authorization | Bearer *String* | yes | API token. |
+
+##### *Body:*
+
+```json5
+{
+  "user_id": 1, // Integer; ID of user
```

Comment:
Since we do ask for both `user_id` and `email` when subscribing user, we should ask for both also here. It will give us the freedom to remove one once we decide which of them should be kept and which should be removed from the system.